NomadAudio/src/Win32/WASAPISharedDriver.cpp

```
@@ -645,6 +645,15 @@ void WASAPISharedDriver::audioThreadProc() {
         BYTE* data = nullptr;
         hr = reinterpret_cast<IAudioRenderClient*>(m_renderClient)->GetBuffer(availableFrames, &data);
         if (FAILED(hr)) {
+            if (hr == AUDCLNT_E_DEVICE_INVALIDATED || 
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)